src/test/Makefile

```diff
@@ -438,11 +441,24 @@ COMMON_FLAGS = \
 	-O0 \
 	-DUNIT_TEST \
 	-isystem $(GTEST_DIR)/inc \
-	-MMD -MP
+	-MMD -MP \
+	-Wno-c99-extensions \
+	-Wno-reorder
 
 CC_VERSION = $(shell $(CC) -dumpversion)
 CXX_VERSION = $(shell $(CXX) -dumpversion)
 
+# Please revisit versions when new clang version arrive. Supported versions: { Linux: 7 - 10; OSX: 7- 12 }
+CC_VERSION_MAJOR := $(shell clang -dumpversion | cut -f1 -d.)
```

Comment:
Wait a minute - we are already dumping right above: https://github.com/betaflight/betaflight/pull/10289/files#diff-3e483a6847052a55659969aafe8a9deef768c53704e0f227d06ffbd9ca024aa0L443

This can be `$(firstword $(subst ., ,$(CC_VERSION)))` for more consistency.

Maybe this will also fix the inconsistent behaviour in travis.

Reply:
Travis still reports 4.2.1. ([Clang tries to pretend to be GCC ?](url)) Changed line above.

Reply:
See also https://groups.google.com/forum/m/#!topic/stan-dev/Js80nYOCnPw